HangMan/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "HangMan"
```

Comment:
This needs to be added to the workspace [here](https://github.com/DioxusLabs/example-projects/blob/master/Cargo.toml#L11) to fix `cargo run`